src/main/kotlin/no/nav/familie/ef/mottak/service/SakService.kt

```diff
@@ -0,0 +1,86 @@
+package no.nav.familie.ef.mottak.service
+
+import no.nav.familie.ef.mottak.config.DOKUMENTTYPE_BARNETILSYN
+import no.nav.familie.ef.mottak.config.DOKUMENTTYPE_OVERGANGSSTØNAD
+import no.nav.familie.ef.mottak.config.DOKUMENTTYPE_SKOLEPENGER
+import no.nav.familie.ef.mottak.integration.IntegrasjonerClient
+import no.nav.familie.ef.mottak.repository.domain.Soknad
+import no.nav.familie.kontrakter.felles.infotrygdsak.OpprettInfotrygdSakRequest
+import no.nav.familie.kontrakter.felles.journalpost.*
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+const val INFOTRYGD = "IT01"
+
+const val FAGOMRÅDE_ENSLIG_FORSØRGER = "EF"
+
+const val SAKSTYPE_SØKNAD = "S"
+
+
+@Service
+class SakService(private val integrasjonerClient: IntegrasjonerClient,
+                 private val søknadService: SøknadService) {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+
+    fun opprettSakOmIngenFinnes(søknadId: String) {
+
+        val soknad = søknadService.get(søknadId)
+
+
+        val journalposterForBrukerRequest = JournalposterForBrukerRequest(Bruker(soknad.fnr, BrukerIdType.FNR),
+                                                                          50,
+                                                                          listOf(Tema.ENF),
+                                                                          listOf(Journalposttype.I))
+        val journalposter = integrasjonerClient.finnJournalposter(journalposterForBrukerRequest)
+
+        val fagsakOpprettet = journalposter.any { it.sak?.fagsaksystem == INFOTRYGD && it.sak?.fagsakId != null }
+
+        if (fagsakOpprettet) {
+            return
+        }
```

Comment:
Vi oppretter kun dersom det ikke finnes noe fra før for hverken OS, BT eller SP - sant? Det er kanskje ikke mulig å plukke ut så spesifikke detaljer fra journalpostsøk/fagsaksøk ?

Reply:
Saksbehandlerne gjør i følge Mirja en vurdering på om det er en ny sak eller kan knyttes til den gamle. Tror det er best å la dem fortsette med det.